Classes/Service/CompressImageService.php

```diff
@@ -164,6 +165,7 @@ public function initializeCompression($file): void
                     $this->addMessageToFlashMessageQueue('success', [0 => (string)$percentageSaved . '%'], FlashMessage::INFO);
                 }
             } catch (\Exception $e) {
+                $this->saveError($file, $e);
```

Comment:
shouldn't we also empty the field compress_error when the compression was successful?

Reply:
right, done (this can happen on replacing a file)